llvm/test/MC/AMDGPU/gfx11_asm_vopc_t16_promote.s

```diff
@@ -12,13 +12,13 @@ v_cmp_class_f16 vcc, vcc_hi, v255
 v_cmp_class_f16 vcc, vcc_lo, v255
 // GFX11: v_cmp_class_f16_e64
 
-v_cmp_class_f16 vcc_lo, v127, v255
+v_cmp_class_f16 vcc, v127, v255
```

Comment:
This was a bug and vcc_lo should have never be allowed as sdst with wave64, but we had both wavefrontsize32 and wvefrontsize64 set, thus it was accepted.